src/applications/vaos/covid-19-vaccine/components/ConfirmationPageV2.jsx

```diff
@@ -87,10 +101,18 @@ function ConfirmationPageV2({
           className="far fa-calendar vads-u-margin-right--1"
         />
         <AddToCalendar
-          summary="VA Appointment"
-          description=""
+          summary={`Appointment at ${clinic?.serviceName}`}
+          description={{
+            text: `You have a health care appointment at ${
+              clinic?.serviceName
+            }`,
+            phone: formatFacilityPhone(facilityDetails),
+            additionalText: [
+              'Sign in to VA.gov to get details about this appointment',
+            ],
+          }}
           location={formatFacilityAddress(facilityDetails)}
-          startDateTime={appointmentDateString}
+          startDateTime={momentDate.format()}
```

Comment:
Not using appointmentDateString due to current format causing an invalid moment object during conversion.